streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/controller/PassportController.java

```diff
@@ -63,6 +68,9 @@ public class PassportController {
     @Autowired
     private Authenticator authenticator;
 
+    @Resource
```

Comment:
It is recommended to use `@Autowired`, because this project uses `@Autowired` uniformly